knexfile.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 
   development: {
     client: 'pg',
-    connection: process.env.DATABASE_URL + '?ssl=true',
+    connection: process.env.DATABASE_URL,
```

Comment:
was supposed to be in a previous commit however I accidentally pushed from the data folder, and didn't notice the knexfile hadn't been added.